main_test.go

```diff
@@ -839,3 +839,17 @@ func TestNoWarmupBenchmark(t *testing.T) {
 	assert.Contains(t, buf.String(), "Total errors: 100")
 	assert.Contains(t, buf.String(), "Error rate: 100")
 }
+
```

Comment:
This test is admittedly a little weak, but should cover the most basic functionality.  @prashantv, open to ideas on how to improve this.